### ISSUE TITLE
my viewController won't execute dealloc method.

### ISSUE THREAD

**Issue by cs13886** [iSofTom/STCollapseTableView#1]
Hi,Thanks for your code first!
I used it in my project.
when i add STCollapseTableView in my ViewController's view, the ViewController will not  start the dealloc method.

```
    _listTableView = [[STCollapseTableView alloc]initWithFrame:CGRectZero style:UITableViewStylePlain];
    [_listTableView setBackgroundColor:[UIColor clearColor]];
    _listTableView.backgroundView = nil;
    _listTableView.delegate = self;
    _listTableView.dataSource = self;
    _listTableView.exclusiveSections = NO;
```

I find that when I  do like this : 
          _listTableView = [[STCollapseTableView alloc]initWithFrame:CGRectZero style:UITableViewStylePlain];
        [_listTableView setBackgroundColor:[UIColor clearColor]];
        _listTableView.backgroundView = nil;
        _listTableView.exclusiveSections = NO;

 delete them :
      _listTableView.delegate = self;
        _listTableView.dataSource = self;

the view controller will start dealloc method

**Comment by iSofTom** [comment]
Ok, I'll investigate that ...

**Comment by iSofTom** [comment]
Fixed in version 0.1.1
